zcutil/build.sh

```diff
@@ -105,15 +97,11 @@ then
     shift
 fi
 
-PREFIX="$(pwd)/depends/$BUILD/"
-
 eval "$MAKE" --version
-eval "$CC" --version
-eval "$CXX" --version
 as --version
 ld -v
 
 HOST="$HOST" BUILD="$BUILD" NO_PROTON="$PROTON_ARG" "$MAKE" "$@" -C ./depends/ V=1
 ./autogen.sh
-CC="$CC" CXX="$CXX" ./configure --prefix="${PREFIX}" --host="$HOST" --build="$BUILD" "$HARDENING_ARG" "$LCOV_ARG" "$TEST_ARG" "$MINING_ARG" "$PROTON_ARG" $CONFIGURE_FLAGS --enable-werror CXXFLAGS='-g'
+CONFIG_SITE="$PWD/depends/$HOST/share/config.site" ./configure "$HARDENING_ARG" "$LCOV_ARG" "$TEST_ARG" "$MINING_ARG" "$PROTON_ARG" $CONFIGURE_FLAGS CXXFLAGS='-g'
```

Comment:
I think this removes ``-Werror`` on all platforms; if so, I don't agree with doing that on Linux.

Reply:
See the corresponding commit message:

> zcutil/build.sh: Remove --enable-werror from default configuration
>
> Windows cross-compile has currently-unavoidable warnings. Upstream Bitcoin Core
> has them as well. For now, let's remove this from the default configuration, and
> add it to the Linux and OSX CI builders (so we effectively still enforce it for
> merged PRs).